Apps/WebClient/src/ClientApp/src/utility/errorTranslator.ts

```diff
@@ -0,0 +1,57 @@
+export default class ErrorTranslator {
+    public static getHumanReadable(errorCode: string): string {
+        let sections = errorCode.split("-");
+        console.log(sections);
+        if (sections.length === 1) {
+            return sections[0];
+        } else if (sections.length === 2) {
+            return (
+                sections[0] +
+                " got an " +
+                this.getErrorType(sections[1]) +
+                " Error."
+            );
+        } else if (sections.length === 3) {
+            return (
+                sections[0] +
+                " got an " +
+                this.getErrorType(sections[1]) +
+                " error while processing a " +
+                this.getServiceName(sections[2]) +
+                " request."
+            );
+        } else {
+            return errorCode;
+        }
+    }
+
+    private static getErrorType(errorType: string): string {
```

Comment:
seems like errorType: string should be errorTypeCode: string instead

Reply:
Just to match the error type code on the backend I left is as errorType